src/app/@ansyn/ansyn/modules/filters/components/filter-container/filter-container.component.ts

```diff
@@ -87,6 +96,7 @@ export class FilterContainerComponent implements OnInit, OnDestroy {
 	ngOnInit() {
 		this.subscribers.push(
 			this.metadataFromState$.subscribe(),
+			this.filterCountersFromState$.subscribe(),
```

Comment:
why not use @AutoSubscription instead of doing all of these subscriptions manually?